ideas.txt

```diff
@@ -0,0 +1,8 @@
+1. How do you prevent multiple "genesis" nodes
+ - publish the hash of the shared secret  to the L1, and also added to the haeder of the rollup?
+ - hardcode the address of the management smart contract and each enclave can decide what to do when it ingests the blocks
+
+2. How to make the secret sharing more efficient?
```

Comment:
If the end goal is for the MC to handle the secret sharing do care about the effiency ? Reckon we can chat about this offline huh ?

Reply:
you wouldn't want multiple aggregators to pay an L1 transaction for no good reason.